src/types/CommentAggregates.ts

```diff
@@ -9,6 +9,4 @@ export interface CommentAggregates {
   downvotes: /* integer */ number;
   published: string;
   child_count: number;
-  hot_rank: number;
-  controversy_rank: number;
```

Comment:
Is there supposed to be a controversial sort type to go along with this controversy rank?

Reply:
Yep, its in `SortType` and `CommentSortType`.